hoon/reference/stdlib/2b.md

```diff
@@ -600,6 +605,27 @@ The accumulator, which is a noun.
     ! exit
 ```
 
+`+mul`'s default sample is 1, so calling `+reel` with `+mul` yields the expected behavior:
+```
+> *mul
+1
+
+> (reel `(list @)`~[1 2 3 4] mul)
+24
+```
+However, if you build a gate that uses `+mul` like so, the sample defaults to 0
+since that is the bunt of `@`:
+```
+> (reel `(list @)`~[1 2 3 4] |=([a=@ b=@] (mul a b)))
+0
+```
+We can fix this with `|:`:
+```
+> (reel `(list @)`~[1 2 3 4] |:([a=`@`1 b=`@`1] (mul a b)))
```

Comment:
Oh so `|:` is just a more explicit `|=  a=_1`? That's kinda neat. The `` `@` `` casts make this feel a little bit clunky, I think for this example it might be fine to not care about these technically becoming `@ud`. But your call!

Reply:
Good call